HOW_TO_CONTRIBUTE.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Adding a dataset to XGI-DATA
+
+### Creating a JSON file for a dataset
+1. Create a script titled `import_<dataset name>.py`. Choose a dataset name that is terse, yet descriptive.
+2. Converting the raw data into an `xgi` hypergraph. Examples of importing are in the [code](/code/) folder.
+3. Save the dataset to a JSON file using the `xgi.write_json()` [method](https://xgi.readthedocs.io/en/stable/api/readwrite/xgi.readwrite.json.html#module-xgi.readwrite.json).
+
+### Adding to Zenodo
+1. Navigate to the [XGI page](https://zenodo.org/communities/xgi) on Zenodo.
+2. Click the "New Upload" button. This should prompt you to log into Zenodo and will bring up the form to upload a new dataset.
+3. Enter the information in the "new upload" form:
+   1. In the "Files" section, drag and drop the file or click the "Upload files" button.
+   2. When asked "Do you already have a DOI for this upload?", select "No".
+   3. Under "Resource type", select "Dataset" from the dropdown list.
+   4. Under "Title" enter the dataset name selected above.
+   5. Under "Creators", add yourself with name, ORCID, and affiliation along with your role (typically "Data collector" or "Data curator")
+   6. Under "Description" write the name of the dataset, where it is from, how it was collected, what nodes and edges are, and some basic statistics about the dataset.
+   7. Under "Version", type "v0.0" if this is the first version of the dataset.
```

Comment:
v0.0 or v0.1?

Reply:
I believe that it is v0.0. At least, that's what I've been doing.